locales/it/index.yml

```
@@ -465,6 +465,7 @@ navigation:
   messages: Messaggi
   profile: Profilo
   accessibility: "Sezione {{section}}, pannello, {{order}} di 4"
+  status: Selezionato
```

```suggestion
  selected: Selezionato
```
The same as the previous comment.

---

Agree! I do it right away